scripts/bootstrap.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+err_report() {
+  echo "Exited with error on line $1"
+}
+trap 'err_report $LINENO' ERR
+
+IFS=$'\n\t'
+
+function grant_cluster_admin {
+  local username="$1"
+  kubectl create clusterrolebinding $username-cluster-admin-binding --clusterrole=cluster-admin --user=$username@gmail.com
+}
+
+function init_argocd {
+  kubectl create namespace argocd
+  kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
```

Comment:
Separate the url for argo:

```suggestion
  local algocd_url = "https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml"
  kubectl apply -n argocd -f $argocd_url
```

Reply:
local is usually used for passing args local scope to that function, either we move this as global variable but less readable, or keep it like this

taken from https://github.com/argoproj/argo-cd/blob/master/docs/getting_started.md#1-install-argo-cd